examples/with-sentry/pages/index.tsx

```diff
@@ -0,0 +1,285 @@
+        alt="Sentry Logo"
+        width={360}
+        height={74}
+        priority
+      />
+      <div
+        className={styles.justCenter}
+        style={{ display: 'flex', flexDirection: 'column', rowGap: '1rem' }}
+      >
+        <p className={inter.className}>
+          See how Sentry records unhandled exceptions in your code.
+        </p>
+        <div className={styles.description}>
+          <p className={inter.className}>
+            <strong>Important:</strong> exceptions in development mode take a
+            different path than in production. These tests should be run on a
+            production build (i.e.{' '}
+            <code className={styles.code}>next build</code>
+            ).{' '}
+            <Link
+              className={styles.link}
+              target="_blank"
+              href="https://nextjs.org/docs/advanced-features/custom-error-page#customizing-the-error-page"
+            >
+              Read more
+            </Link>
+          </p>
+        </div>
+      </div>
+      <br />
+      <div className={styles.grid}>
+        <div className={styles.card}>
+          <h2 className={inter.className}>
+            API routes <span>-&gt;</span>
+          </h2>
+          <p className={inter.className}>
+            The following examples are Sentry tests. Note that 1 and 2 are not
+            expected to work if deployed to Vercel yet.
+          </p>
+          <p className={inter.className}>
+            Top-of-module promise that rejects, but its result is not awaited.
+            {'  '}
+            <code className={styles.code}>
+              <a className={styles.link} href="/api/test1" target="_blank">
+                API Test 1
+              </a>
+            </code>
+          </p>
+          <p className={inter.className}>
+            API has a top-of-module exception.{' '}
+            <code className={styles.code}>
+              <a className={styles.link} href="/api/test2" target="_blank">
+                API Test 2
+              </a>
+            </code>
+          </p>
+          <p className={inter.className}>
+            API with an exception in its request handler.{' '}
+            <code className={styles.code}>
+              <a className={styles.link} href="/api/test3" target="_blank">
+                API Test 3
+              </a>
+            </code>
+          </p>
+          <p className={inter.className}>
+            API uses a try/catch to handle an exception and records it.{' '}
+            <code className={styles.code}>
+              <a className={styles.link} href="/api/test4" target="_blank">
+                API Test 4
+              </a>
+            </code>
+          </p>
+        </div>
+        <div className={styles.card}>
+          <h2 className={inter.className}>
+            SSR <span>-&gt;</span>
+          </h2>
+          <p className={inter.className}>
+            There are currently two known bugs with respect to SSR transactions:
```

Comment:
Could this be more clear on where the issue arises here? It sounds like a Vercel bug, is it?